src/constructs/aws/StaticWebsite.ts

```diff
@@ -278,39 +279,26 @@ export class StaticWebsite extends AwsConstruct {
         return this.provider.getStackOutput(this.distributionIdOutput);
     }
 
-    private errorResponse(): ErrorResponse {
+    private errorResponseDocument(): string {
         // Custom error page
         if (this.configuration.errorPage !== undefined) {
-            let errorPath = this.configuration.errorPage;
+            const errorPath = this.configuration.errorPage;
             if (errorPath.startsWith("./") || errorPath.startsWith("../")) {
                 throw new ServerlessError(
                     `The 'errorPage' option of the '${this.id}' static website cannot start with './' or '../'. ` +
                         `(it cannot be a relative path).`,
                     "LIFT_INVALID_CONSTRUCT_CONFIGURATION"
                 );
             }
-            if (!errorPath.startsWith("/")) {
-                errorPath = `/${errorPath}`;
-            }
```

Comment:
This bit was removed (and the corresponding test changed): this isn't necessary anymore with S3 doing the logic?

Reply:
Exactly, the correct path is resolved with or without a leading `/`